lib/rspec/sidekiq/matchers/have_enqueued_job.rb

```diff
@@ -31,7 +31,7 @@ def at_evaluator(value)
 
         def in_evaluator(value)
           return false if job['at'].to_s.empty?
-          (Time.now + value).to_i == Time.at(job['at']).to_i
+          (Time.now.to_f + value.to_f).to_i == Time.at(job['at']).to_i
```

Comment:
📓 Just for other wondering where Sidekiq does this [calculation](https://github.com/sidekiq/sidekiq/blob/7-1/lib/sidekiq/job.rb#L254-L262). It's also been the same calculation since at least [5.0](https://github.com/sidekiq/sidekiq/blob/5-0/lib/sidekiq/worker.rb#L56-L66).